content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
+
+To check your marker matches one of these types, you can use the `in` operator:
+
+```python
+marker = r.camera.see()[0]  # Only the closest marker
+
+marker.id in WALL
+>>> True 
+```
+
+To see the full range of constants, and the exact IDs they match up to, have a look at [the source](https://github.com/sourcebots/robot-api/blob/master/robot/game_specific.py).
+
+### _Wall_ Markers
+These markers are the ones positioned around the arena, there are 28 in total. Each token has a unique ID, and its location is documented in [the rules](/rules). This may be useful to calculate your position in the arena.
+
+```python
+marker.id in WALL  # Is this marker a wall marker
+```
+
+{{% notice tip %}}
+It's recommended to use `marker.is_wall_marker()` instead of `marker.id in WALL`
```

Comment:
This feels backwards to me -- if this is the recommended way to do this, then we should prioritise documenting that. I'd even argue that if using the helper is the recommended way then we shouldn't document the constant at all. 

This somewhat follows the logic of "There should be one, and preferably only one, obvious way to do it". (Unfortunately taking that to its logical extreme suggests that the helper shouldn't be there at all and we should just use (only) the constants since that would be consistent and thus obvious.)

Reply:
I'm going to leave this in for now. I agree that removing the helper function is the nicest approach, but that probably warrants further discussion

Reply:
:+1: for deprecating `is_wall_marker` for consistency.

Carded up [here](https://trello.com/c/tHwRNdfo/63-deprecate-remove-iswallmarker-istokenmarker-methods).

Reply:
I feel quite strongly that we shouldn't introduce something and then immediately say that something else is the recommended way to do things. I'm happy to postpone actually changing the API, but I think that we should resolve which is the recommended solution before documenting it.

Reply:
I'll drop this recommendation, and we can discuss dropping the helper in the card